mmv1/products/spanner/BackupSchedule.yaml

```diff
@@ -154,3 +154,27 @@ properties:
       - 'incrementalBackupSpec'
     properties:
       []
+  - name: 'encryptionConfig'
+    type: NestedObject
+    description: |
+      Configuration for the encryption of the backup schedule.
+    default_from_api: true
+    properties:
+      - name: 'encryptionType'
+        type: Enum
+        description: |
+          The encryption type of backups created by the backup schedule.
+          Possible values are USE_DATABASE_ENCRYPTION, GOOGLE_DEFAULT_ENCRYPTION, or CUSTOMER_MANAGED_ENCRYPTION.
+          If you use CUSTOMER_MANAGED_ENCRYPTION, you must specify a kmsKeyName.
+          If your backup type is incremental-backup, the encryption type must be GOOGLE_DEFAULT_ENCRYPTION.
+        enum_values:
+          - 'ENCRYPTION_TYPE_UNSPECIFIED'
```

Comment:
We can remove this, there is no reason for a user to specify this value

Reply:
Done